srlife/damage.py

```diff
@@ -235,18 +391,117 @@ def calculate_element_log_reliability(
           material:       material model  object with required data
         """
 
-        svals = material.strength(temperatures)
-        mvals = material.modulus(temperatures)
-
         # Average normal tensile stress raied to exponent mv
-        avg_nstress = self.calculate_avg_normal_stress(
-            mandel_stress, mvals, self.nalpha, self.nbeta
+        avg_nstress = (
+            self.calculate_avg_normal_stress(
+                mandel_stress, self.mvals, self.temperatures, self.material
+            )
+        ) ** (1 / self.mvals)
+
+        return -self.kpvals * (avg_nstress**self.mvals) * volumes
+
+
+class MTSModelGriffithFlaw(CrackShapeDependent):
+    """
+    Maximum tensile stess failure model with a Griffith flaw
+
+    Assigning default values for nalpha and nbeta
+    """
+
+    def calculate_eq_stress(self, mandel_stress, mvals, temperatures, material):
+        """
+        Calculate the equivalent stresses given the pricipal stresses
+        """
+
+        # Projected equivalent stress
+        return 0.5 * (self.sigma_n + np.sqrt((self.sigma_n**2) + (self.tau**2)))
+
+
+class MTSModelPennyShapedFlaw(CrackShapeDependent):
+    """
+    Maximum tensile stess failure model with a penny shaped flaw
+
+    Assigning default values for nalpha and nbeta
+    """
+
+    def calculate_eq_stress(self, mandel_stress, mvals, temperatures, material):
+        """
+        Calculate the average normal tensile stresses given the pricipal stresses
+        """
+        # Projected equivalent stress
+        return 0.5 * (
```

Comment:
Same sort of thing: recalculate sigma_n and tau here and use `material` to get `nu`